development/playbooks/deploy-dev/deploy-dev.yaml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Deploy Foreman Development Environment
+  hosts: "{{ target_host if target_host is defined and target_host != '' else 'quadlet' }}"
+  become: true
+  vars_files:
+    - "../../../src/vars/defaults.yml"
+    - "../../../src/vars/{{ certificate_source }}_certificates.yml"
+    - "../../../src/vars/images.yml"
+    - "../../../src/vars/database.yml"
+    - "../../../src/vars/foreman.yml"
+    - "../../../src/vars/base.yaml"
+  vars:
+    httpd_foreman_backend: "http://localhost:3000"
+  roles:
+    - pre_install
+    - role: certificates
+    - role: postgresql
+      vars:
+        postgresql_databases:
+          - name: "{{ candlepin_database_name }}"
+            owner: "{{ candlepin_database_user }}"
+          - name: "{{ foreman_development_database_name }}"
+            owner: "{{ foreman_database_user }}"
+          - name: "{{ foreman_development_database_name }}_test"
+            owner: "{{ foreman_database_user }}"
+          - name: "{{ pulp_database_name }}"
+            owner: "{{ pulp_database_user }}"
+        postgresql_users:
+          - name: "{{ candlepin_database_user }}"
+            password: "{{ candlepin_database_password }}"
+          - name: "{{ foreman_database_user }}"
+            password: "{{ foreman_database_password }}"
+            role_attr_flags: SUPERUSER
```

Comment:
where is this applied?

Reply:
From my testing, it was being applied magically... but I've made it more explicit.